src/quota_aggregator_impl.cc

```diff
@@ -125,7 +129,8 @@ ::google::protobuf::util::Status QuotaAggregatorImpl::Quota(
     // requests will be aggregated to this temporary element until the
     // response for the actual request arrives.
     ::google::api::servicecontrol::v1::AllocateQuotaResponse temp_response;
-    CacheElem* cache_elem = new CacheElem(temp_response);
+    CacheElem* cache_elem = new CacheElem(request, temp_response,
+                                          SimpleCycleTimer::Now());
     cache_elem->set_signature(request_signature);
     cache_elem->set_in_flight(true);
     cache_->Insert(request_signature, cache_elem, 1);
```

Comment:
When the cached response is negative, exceeded timeout and quota was refilled; the client request will be rejected until the response arrives. Would this be ok?